src/AlgebraicGeometry/Schemes/Covering/Objects/Methods.jl

```diff
@@ -68,6 +69,25 @@ function update_glueing_graph(C::Covering)
   return gg
 end
 
+## prune the covering by throwing away empty charts and then compute
+## the glueing graph afterwards 
+## (relevant data for connectedness of glueing)
+function pruned_glueing_graph(C::Covering)
+  v = findall(U->!is_empty(U), C.patches)
+  m = length(v)
+  gt = Graph{Undirected}(m)
+  for (X, Y) in keys(glueings(C))
+    i = findfirst(==(C[X]),v)
+    !isnothing(i) || continue
+    j = findfirst(==(C[Y]),v)
+    !isnothing(j) || continue
+    (U, V) = glueing_domains(C[X,Y])
+    is_dense(U) && add_edge!(gt, i, j)
+    is_dense(V) && add_edge!(gt, j, i)
```

Comment:
what happens here? shouldn't we just extract the subgraph consisting of non-empty glueings? 
Then why do we ask for something to be dense. Probably I do not understand the purpose of this function.

Reply:
This line is from the original version, which I only changed to handle empty patches properly. I understand it as a sanity check that we are really glueing along opens.